wallets/provider-trezor/src/helpers.ts

```diff
@@ -20,6 +20,9 @@ export async function getTrezorModule() {
 }
 
 export function getTrezorErrorMessage(error: any) {
+  if (error?.message === 'Cancelled') {
```

Comment:
Please check if you can use `cleanEVMError` function.

Reply:
I had used `cleanEVMError` but it didn't support this error

Reply:
```
export function getTrezorErrorMessage(error: any) {
  if (error?.message === 'Cancelled') {
    return new Error('User rejected the transaction.', { cause: error });
  }
  if (error?.shortMessage) {
    /*
     * Some error signs have lengthy, challenging-to-read messages.
     * shortMessage is used because it is shorter and easier to understand.
     */
    return new Error(error.shortMessage, { cause: error });
  }
  return cleanEvmError(error);
}
```

Reply:
`cleanEvmError` should support errors thrown by ethers and provider, we should not handle a single error using an `if` statement. Please check the type of error implement and see if there are other errors which are not supported properly and implement a general solution.

Reply:
Based on the error we received from the trezor side, I modified this section.

 Trezor error type:

```
export interface Unsuccessful {
    success: false;
    payload: {
        error: string;
        code?: string;
    };
}

```